internal/operands/common-instancetypes/reconsile_test.go

```diff
@@ -0,0 +1,92 @@
+package common_instancetypes
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	ssp "kubevirt.io/ssp-operator/api/v1beta1"
+	"kubevirt.io/ssp-operator/internal/common"
+	"kubevirt.io/ssp-operator/internal/operands"
+	. "kubevirt.io/ssp-operator/internal/test-utils"
+)
+
+var log = logf.Log.WithName("common-instancetypes-operand")
+
+func TestInstancetypes(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Common Instance types Suite")
+}
+
+var _ = Describe("Common-Instancetypes operand", func() {
+
+	var (
+		err     error
+		operand operands.Operand
+		request common.Request
+	)
+
+	const (
+		namespace        = "kubevirt"
+		name             = "test-ssp"
+		instancetypePath = "../../../" + BundleDir + ClusterInstancetypesBundlePrefix + ".yaml"
```

Comment:
Could this become a problem in d/s tests? Dependency on paths?

Reply:
I wouldn't think so, the bundles should be static in the tree even downstream.

Reply:
ACK